contracts/Rollup.sol

```diff
@@ -352,7 +352,7 @@ contract Rollup is Ownable, RollupHelpers, RollupInterface {
      */
     function withdraw(
         uint64 idBalanceTree,
-        uint16 amount,
+        uint256 amount,
```

Comment:
LoadAmount is 128bits (deposit, transfer) and this 256bits
I don't know if there's a reason

Reply:
- maximum deposit value is `2**128`. however, values could be added between them reaching values above `2**128`. Here we set a limit, inside the circuit, of `2**192` as overflow.

- Given that we limit the deposits to `2**128`, in order to reach values above `2**192`, users must submit at least `2**64` deposit transactions to reach that overflow. Assuming an on-chain fee of `0.01 ETH`, users should spend `2**64 * 0.01 ETH =   1.84e+17` which seems not feasible